daemon/execdriver/native/template/default_template_linux.go

```diff
@@ -40,7 +37,8 @@ func New() *configs.Config {
 			{Type: "NEWUSER"},
 		}),
 		Cgroups: &configs.Cgroup{
-			Parent: "/docker",
+			Parent:      "/docker",
```

Comment:
Maybe there is no point we set it to "/docker" here anymore, since we will always override it, and it feels wired to set `Parent=/docker` and `ScopePrefix=docker` in the same time and same place, because they will never work together.
What about just set `Parent: ""` here?

And one more comment need to be fixed:
```
diff --git a/daemon/execdriver/driver_unix.go b/daemon/execdriver/driver_unix.go
index eedd8ac..b2a3d30 100644
--- a/daemon/execdriver/driver_unix.go
+++ b/daemon/execdriver/driver_unix.go
@@ -150,7 +150,7 @@ func InitContainer(c *Command) *configs.Config {
        // check to see if we are running in ramdisk to disable pivot root
        container.NoPivotRoot = os.Getenv("DOCKER_RAMDISK") != ""

-       // Default parent cgroup is "docker". Override if required.
+       // Override it if required.
        if c.CgroupParent != "" {
                container.Cgroups.Parent = c.CgroupParent
        }
```